.travis.yml

```diff
@@ -64,7 +64,7 @@ script:
     # so this should work fine most of the time, and be a lot better than not
     # checking at all.
     - git diff -U20 HEAD~40 |
-        filterdiff -x "src/jtag/drivers/libjaylink/*" -x "tools/git2cl/*" |
+        filterdiff -x "b/src/jtag/drivers/libjaylink/*" -x "b/tools/git2cl/*" |
```

Comment:
Did libjaylink and git2cl move or does the "b/..." mean something special?

Reply:
They didn't move. This line probably never worked.
In git diff output it shows the original version with a path of a/..., and the new version with a path of b/.... filterdiff filters on the new path, so we need to mention b/... in the paths we give it.